static/canvas.js

```diff
@@ -151,38 +151,25 @@ class Canvas {
             myState.valid = false;
         }, true);
 
-        /* Shifts through selection of shapes using spacebar. */
```

Comment:
The spacebar shifting was originally there in case a smaller object was behind a larger one and thereby making it unselectable. I'm not sure how likely it'll occur, but an alternate to this scenario is a list of clickable things that will bring it to the top.